azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -107,6 +104,8 @@
   // 12 weeks
   private static final long DEFAULT_EXECUTION_LOGS_RETENTION_MS = 3 * 4 * 7
       * 24 * 60 * 60 * 1000L;
+  // 10 mins recently finished threshold.
+  private long recentlyFinishedLifetimeMs = 600000;
```

Comment:
Thanks @chengren311 ! 
@jamiesjc please take a look at the `Duration` class for storing time constants.